src/documents/serialisers.py

```diff
@@ -441,6 +441,17 @@ class Meta:
             "data_type",
         ]
 
+    def validate(self, attrs):
+        # see https://github.com/encode/django-rest-framework/issues/7173
```

Comment:
Could you make this a TODO maybe, so it's easier to find when (if) 3.15 is released?

Reply:
Sure yea! Took the liberty of changing the other one too